src/utils.c

```diff
@@ -49,6 +49,55 @@ object_get_name (GObject *object)
   return NULL;
 }
 
+void
+object_emit_action_signal (GApplication *app,
+                           GObject      *object,
+                           GSignalQuery *signal,
+                           const gchar  *detail,
+                           GVariant     *params,
+                           GError      **error)
+{
+  g_auto(GValue) retval = G_VALUE_INIT;
+  g_autoptr(GVariant) variant = NULL;
+  g_autoptr(GArray) values = NULL;
+  GValue *instance_and_params;
+  guint i;
```

Comment:
You should declare this in the loop declaration itself.

Reply:
C89